security/certs.go

```diff
@@ -129,7 +142,7 @@ func RunCreateCACert(certsDir string, keySize int) error {
 // RunCreateNodeCert is the entry-point from the command-line interface
 // to generate node certs and keys:
 // - node.server.{crt,key}: server cert with list of dns/ip addresses
-// - node.{crt,key}: client cert with "node" as the Common Name.
+// - node.client.{crt,key}: client cert with "node" as the Common Name.
```

Comment:
my next step (or somewhere down the line at least) in all this is to extract the username from the certificate and pass it down to whoever is using it. That whoever should have a requested username somewhere. Most of this is not passed around right now, but it needs to be.